common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -418,6 +421,46 @@ def assertGroupContent(self, item, title=None, notes=None):
         self.assertEqual(item.title, title)
         self.assertEqual(item.notes, notes)
 
+    def assert_viewed_event(self, view=None):
+        """
+        Verifies that the correct view event was captured for the Notes page.
+        """
+        # There will always be an initial event for "Recent Activity" because that is the default view.
+        actual_events = self.wait_for_events(
+            event_filter={'event_type': 'edx.course.student_notes.notes_page_viewed'},
+            number_of_matches=1 if view is None else 2
```

Comment:
My intention was that you don't pass in the view if it should just be "Recent Activity". But I can put in a simple check for that case so that the method is robust to whether or not "Recent Activity" is passed.